### PR TITLE
Sciml cleanup

### PR DESCRIPTION
**Pull request by Tom-Szendrey**
# Description
Removing sciml MQ and related types ect from teraruim
Before removing the rest of sciml (application.properties, docker file, simulation service proxy) we will need to replace the model-to-latex utilized in transform controller

# Note
to keep focused PRs I will have a second PR replacing this model-to-latex as well as removing the rest of sciml

### CODE REVIEW

File: packages/client/hmi-client/src/services/model.ts [hunk 1]
```diff
@@ -175,24 +175,6 @@ export function getModelType(model: Model | null | undefined): AMRSchemaNames {
...
-
-	/* TODO - Replace the GET with the POST when the backend is ready,
-	 *        see PR https://github.com/DARPA-ASKEM/sciml-service/pull/167
-	 */
-	const response = await API.get(`/transforms/model-to-latex/${model.id}`);
-	// const response = await API.post(`/transforms/model-to-latex/`, model);
-	const latex = response?.data?.latex;
-	if (!latex) return '';
-	return latex ?? '';
-}
```
Comment (mwdchang) [inline packages/client/hmi-client/src/services/model.ts:194]: Note @Tom-Szendrey until we have a replacement for `model-to-latex` functionality, which maybe @YohannParis is evaluating, this is "clean up SciML", not "remove SciML".

Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/services/model.ts:194]: I was just double checking this actually! 
Good to know I will add this and its requirements back

Comment (mwdchang) [reply packages/client/hmi-client/src/services/model.ts:194]: See https://github.com/DARPA-ASKEM/terarium/issues/4090

Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/services/model.ts:194]: ahh perfect! Thanks Dan

**Review by YohannParis — APPROVED** [review]
LGTM

**Review by mwdchang — APPROVED** [review]

**Pull request merged**